wls-gui-wahllokalsystem/tests/components/vorfaelleundvorkommnisse/TheEreignisseRow.spec.ts

```diff
@@ -201,5 +201,37 @@ describe("TheEreignisseRow.vue", () => {
         0
       );
     });
+
+    it("should_triggerUpdateUhrzeitInStore_when_uhrzeitOfEreignisWasChanged", async () => {
+      const ereignisStore = useEreignisStore();
+
+      const ereigniseintraege = [] as Ereignis[];
+      for (let i = 0; i < 5; i++) {
+        const date = new Date();
+        date.setHours(i, 0);
+        ereigniseintraege.push(
+          EreignisBuilder.createComplete()
+            .withUhrzeit(date)
+            .withBeschreibung(`Vorfall Nr.: ${i}`)
+        );
+      }
+
+      ereignisStore.wahlbezirkEreignisse.ereigniseintraege = ereigniseintraege;
+
+      await nextTick();
+
+      const indexOfTimeInputForChange = 3;
+      const firstEreignisTimeinput = wrapper.findAllComponents(
+        '[data-test="baseTimeInput"]'
+      )[indexOfTimeInputForChange];
```

Comment:
ich finde wir sollten variablen die nur einmal wiederverwendet werden vermeiden. sie sind zwar besser lesbar, aber ich finde nicht dass der code dadurch übersichtlicher wird, weil man immer erst mal schauen muss welcher wert da dahinter steckt.
```suggestion
      const firstEreignisTimeinput = wrapper.findAllComponents(
        '[data-test="baseTimeInput"]'
      )[3];
```

Reply:
`indexOfTimeInputForChange ` wird 2 mal verwendet:
- `const firstEreignisTimeinput = wrapper.findAllComponents('[data-test="baseTimeInput"]');`
- `expect(ereignisStore.updateUhrzeitByIndex).toHaveBeenCalledWith(newValue, indexOfTimeInputForChange);`